doc/api/util.md

```diff
@@ -1989,6 +1989,53 @@ const channel = new MessageChannel();
 channel.port2.postMessage(signal, [signal]);
 ```
 
+## `util.aborted(signal, resource)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `signal` {AbortSignal}
+* `resource` {Object} Any non-null entity, reference to which is held weakly.
+* Returns: {Promise}
+
+Listens to abort event on the provided `signal` and
+returns a promise that resolves when the `signal` is
+aborted. The function is dependent on the `resource` and
+the returned promise shall remain pending if `resource`
+goes out of memory. The function shall automatically
+cleanup any event listeners it attaches to `signal`.
```

Comment:
"the function is dependent" and "the function shall cleanup" is a bit unclear which function we're talking about. I think the fact that we're using a event listener is more an implementation detail than something we should document, and if anything the fact that we clean up goes without saying, so I would remove the sentence altogether.

```suggestion
aborted. If the passed `resource` is garbage collected,
the returned promise shall remain pending indefinitely.
```

Reply:
Ah yes went a little to verbose 😅